packages/cdk/lib/env/context-app-parameters.ts

```diff
@@ -212,4 +218,28 @@ export class ContextAppParameters {
     }
     return defFilesystem;
   }
+
+  public getEngineType(): string {
+    let engineType: string;
+    switch (this.engineName) {
+      case "cromwell":
+        engineType = "wdl";
+        break;
+      case "nextflow":
+        engineType = "nextflow";
+        break;
+      case "miniwdl":
+        engineType = "wdl";
+        break;
+      case "snakemake":
+        engineType = "snakemake";
+        break;
+      case "toil":
+        engineType = "cwl";
+        break;
+      default:
+        throw Error(`Engine '${this.engineName}' is not supported`);
```

Comment:
Does it need to be blocking? Just a log with a warn or a “unknown” value should be enough. Thoughts?

Reply:
Here just following the pattern of `getDefaultFilesystem()` above, which throws the error. For me, I think it's fine to leave as is.

Reply:
Tagging the resources is for our analytics and not a "customer use case". So, I we should not block a customer use case (stack deployment in this case) when we fail to associate analytical data.

Reply:
Yes I agree.